### PR TITLE
Add bluetooth c sample

### PR DESCRIPTION
**Pull request by larissa-shinohara**

### CODE REVIEW

File: bluetooth/c/.conf/.container
Comment (andreriesco) [inline bluetooth/c/.conf/.container:1]: Please create an initial project called `bluetoothconnection`

Comment (larissa-shinohara) [reply bluetooth/c/.conf/.container:1]: Thanks, André. We did it now!

File: bluetooth/c/.vscode/settings.json [hunk 1]
```diff
@@ -1,19 +1,27 @@
 {
-  "torizon_psswd": "",
-  "torizon_login": "",
-  "torizon_ip": "",
-  "host_ip": "",
+  "torizon_psswd": "1",
```
Comment (andreriesco) [inline bluetooth/c/.vscode/settings.json:2]: Please remove this settings here, keep it without values

File: bluetooth/c/.vscode/settings.json [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
+  "host_ip": "10.22.1.115",
   "torizon_workspace": "${workspaceFolder}",
   "torizon_debug_port": "",
   "torizon_debug_ssh_port": "2230",
   "torizon_debug_port2": "",
   "terminal.integrated.defaultProfile.linux": "bash",
   "torizon_debug_port3": "",
-  "torizon_gpu": "",
-  "torizon_arch": "",
+  "torizon_gpu": "-vivante",
```
Comment (andreriesco) [inline bluetooth/c/.vscode/settings.json:12]: Also here

File: bluetooth/c/.vscode/settings.json [hunk 3]
```diff
@@ -1,19 +1,27 @@
...
+  "torizon_arch": "aarch64",
   "wait_sync": "1",
   "torizon_run_as": "torizon",
   "torizon_app_root": "/home/torizon",
   "tcb.packageName": "bluetoothconnection",
-  "tcb.version": "early-access"
+  "tcb.version": "early-access",
+  "docker.environment": {
```
Comment (andreriesco) [inline bluetooth/c/.vscode/settings.json:19]: And also this `docker.environment` variable

File: bluetooth/c/Dockerfile [hunk 4]
```diff
@@ -22,8 +22,14 @@ ENV IMAGE_ARCH ${IMAGE_ARCH}
 # __deps__
 RUN apt-get -q -y update && \
     apt-get -q -y install \
+    bluez \
+    libc6-dev \
+    libbluetooth-dev \
 # DOES NOT REMOVE THIS LABEL: this is used for VS Code automation
     # __torizon_packages_dev_start__
+	bluez:arm64 \
```
Comment (andreriesco) [inline bluetooth/c/Dockerfile:30]: Also here this values can be removed, as the values listed on the `torizonPackages.json` get added automatically by the extension here

File: bluetooth/c/Dockerfile [hunk 5]
```diff
@@ -55,6 +61,9 @@ ENV APP_EXECUTABLE ${APP_EXECUTABLE}
 RUN apt-get -y update && apt-get install -y --no-install-recommends \
 # DOES NOT REMOVE THIS LABEL: this is used for VS Code automation
     # __torizon_packages_prod_start__
+	bluez:arm64 \
```
Comment (andreriesco) [inline bluetooth/c/Dockerfile:64]: Also here

File: bluetooth/c/Dockerfile.debug [hunk 6]
```diff
@@ -63,6 +63,9 @@ RUN apt-get -q -y update && \
     apt-get -q -y install \
 # DOES NOT REMOVE THIS LABEL: this is used for VS Code automation
     # __torizon_packages_dev_start__
+	bluez:arm64 \
```
Comment (andreriesco) [inline bluetooth/c/Dockerfile.debug:66]: And here

**Comment by andreriesco** [comment]
Also, please squash commits [98033577bbcf75c3aa28d54cb36a784335bc311b](https://github.com/toradex/torizon-samples/pull/7/commits/98033577bbcf75c3aa28d54cb36a784335bc311b) and [98033577bbcf75c3aa28d54cb36a784335bc311b](https://github.com/toradex/torizon-samples/pull/7/commits/98033577bbcf75c3aa28d54cb36a784335bc311b) into one commit. 
And add the template metadata on the `samples.json` file, like in [here](https://github.com/toradex/torizon-samples/pull/6/commits/02851009b472fd982f2d6d6ab42ae7d0608dc69d)

**Pull request closed without merging**